test-cases/tests/upgrade/n07-verify-rds-and-elasticache-db-instance-types-are-successfull.md

```diff
@@ -31,24 +33,36 @@ oc login --token=<TOKEN> --server=https://api.<CLUSTER_NAME>.s1.devshift.org:644
 aws rds describe-db-instances | jq -r '.DBInstances[] | .DBInstanceClass'
 ```
 
-> output should contain only "db.t2.small"
+> output should contain only "db.t3.small"
 
 ```bash
 # Elasticache
 aws elasticache describe-cache-clusters | jq -r '.CacheClusters[] | .CacheNodeType'
 ```
 
-> output should contain only "cache.t2.micro"
+> output should contain only "cache.t3.micro"
 
-3. Follow [the guide for updating the maintenance window](https://github.com/RHCloudServices/integreatly-help/blob/master/sops/2.x/cssre_info/info_aws_update_backup_maintenance_window.md) and set it to an hour from now
-4. Come back after maintenance actions are completed
-5. Check updated RDS and Elasticache instance types with AWS CLI (use the same commands as before)
-   > output for RDS instances should contain only "db.t3.small"
-   > output for Elasticache instances should contain only "cache.t3.micro"
-6. Verify the correct Redis/Elasticache engine version:
+3. Update config map `cloud-resources-aws-strategies` to modify the node types for rds and elastic cache.
+For rds, change the dbInstanceClass to "db.t3.large". For elastic cache change the cachenodetype to "cache.m3.medium"
+4. Follow [the guide for updating the maintenance window](https://github.com/RHCloudServices/integreatly-help/blob/master/sops/2.x/cssre_info/info_aws_update_backup_maintenance_window.md) and set it to an hour from now
+5. Come back after maintenance actions are completed
+6. Check updated RDS and Elasticache instance types with AWS CLI (use the same commands as before)
+   > output for RDS instances should contain only "db.t3.large"
+   > output for Elasticache instances should contain only "cache.m3.medium"
+7. Verify the correct Redis/Elasticache engine version:
 
 ```bash
 aws elasticache describe-cache-clusters | jq -r '.CacheClusters[].EngineVersion'
 ```
 
 > output should contain only the versions "5.0.6"
+
+
+
+
+
+
+
+
+
+
```

Comment:
Possibly no need for so many empty lines at the end. I think the linter will only pass if there is one empty line at the end of the file